jrt/tests/com/yahoo/jrt/InvokeErrorTest.java

```diff
@@ -157,4 +160,36 @@ public void testConnectionError() {
         assertEquals(ErrorCode.CONNECTION, req1.errorCode());
     }
 
+    @org.junit.Test
+    public void testFilterIsInvoked() {
+        Request r = new Request("test");
+        r.parameters().add(new Int32Value(42));
+        r.parameters().add(new Int32Value(0));
+        r.parameters().add(new Int8Value((byte)0));
+        assertFalse(filter.invoked);
+        target.invokeSync(r, timeout);
+        assertFalse(r.isError());
+        assertTrue(filter.invoked);
+    }
+
+    @org.junit.Test
+    public void testFilterFailsRequest() {
+        Request r = new Request("test");
+        r.parameters().add(new Int32Value(42));
+        r.parameters().add(new Int32Value(0));
+        r.parameters().add(new Int8Value((byte)0));
+        filter.allowed = false;
+        assertFalse(filter.invoked);
+        target.invokeSync(r, timeout);
+        assertTrue(r.isError());
+        assertTrue(filter.invoked);
```

Comment:
Should we add an explicit check that the `test` method itself is _not_ invoked when the filter fails? (and vice versa in the case where the RPC passes the authz filter)